misc/snapshotter/config-fscache.toml

```diff
@@ -0,0 +1,22 @@
+version = 1
```

Comment:
Can we reuse the `misc/snapshotter/config.toml` instead of creating a new one?

Reply:
Fixed.